spec/datadog/tracing/contrib/rails/support/rails3.rb

```diff
@@ -1,7 +1,5 @@
 require 'rails/all'
 
-require 'ddtrace' if ENV['TEST_AUTO_INSTRUMENT'] == true
-
 if ENV['USE_SIDEKIQ']
```

Comment:
Should `USE_SIDEKIQ` be cleaned up as well? Right now it's kinda weird that it accepts anything, e.c. `USE_SIDEKIQ=false` will enabled it.

Reply:
That seems like a reasonable ask.